easybuild/easyconfigs/t/tabixpp/tabixpp-1.1.0-GCC-9.3.0.eb

```diff
@@ -0,0 +1,38 @@
+# Author: Jasper Grimm (UoY)
+
+easyblock = 'ConfigureMake'
+
+name = 'tabixpp'
+version = '1.1.0'
+
+homepage = 'https://github.com/ekg/tabixpp'
+description = """C++ wrapper to tabix indexer"""
+
+toolchain = {'name': 'GCC', 'version': '9.3.0'}
+
+github_account = 'ekg'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-1.1.0_use-external-HTSlib.patch']
+checksums = [
+    '56c8f1b07190aba5e1d0b738e380e726d380f0ad8b2d0df133200b0ab1f8ed88',  # v1.1.0.tar.gz
+    '65496b6e042cf54290386a33cf0de26cb1160b3ddb35216765999d790b99cab4',  # tabixpp-1.1.0_use-external-HTSlib.patch
+]
+
+builddependencies = [('binutils', '2.34')]
+
+dependencies = [
+    ('HTSlib', '1.10.2'),
+    ('PCRE', '8.44'),
```

Comment:
I had a quick look at my build, and it seems likes this library links to 
* XZ (liblzma)
* bzip2 (libbz2)
* zlib (libz)

Please add this ad deps